src/Raven.Client/Documents/Session/SessionOptions.cs

```diff
@@ -27,19 +37,33 @@ public enum ShardedBatchBehavior
         NonTransactionalMultiBucket
     }
 
+    /// <summary>
+    /// Configure the Session's behavior
+    /// </summary>
     public sealed class SessionOptions
     {
+        /// <summary>
+        /// Specify session's database, default value is taken from <see cref="IDocumentStore.Database"/>.
+        /// </summary>
         public string Database { get; set; }
 
+        /// <summary>
+        /// Disable tracking for all entities in the session<br/>
+        /// </summary>
+        /// <remarks><seealso href="https://ravendb.net/docs/article-page/6.0/csharp/client-api/session/configuration/how-to-disable-tracking#disable-entity-tracking"/></remarks>
         public bool NoTracking { get; set; }
 
+        /// <summary>
+        /// Disable caching of HTTP responses for the session<br/>
+        /// </summary>
+        /// <remarks><seealso href="https://ravendb.net/docs/article-page/6.0/csharp/client-api/session/configuration/how-to-disable-caching"/></remarks>
         public bool NoCaching { get; set; }
 
         public RequestExecutor RequestExecutor { get; set; }
 
         /// <summary>
-        /// Once TransactionMode set 'ClusterWide' it  will perform the SaveChanges as a transactional cluster wide operation.
-        /// Any document store or delete will be part of this session's cluster transaction.
+        /// Define the transaction mode of the session. <br/>
+        /// Each <see cref="TransactionMode"/> offers a different <seealso href="https://ravendb.net/docs/article-page/6.0/csharp/client-api/faq/transaction-support">isolation and consistency guarantees</seealso>
```

Comment:
incorrect linking, avoid inline urls, this was already solved by @arekpalinski @maciejaszyk in their PRs

Reply:
fixed